src/csm_testing/tests/sat_functional/showrev/test_showrev.py

```diff
@@ -180,7 +180,7 @@ def test_showrev_products_command(self):
                 kubectl_product_names.add(key)
 
         kubectl_product_count = len(kubectl_product_names)
-        self.assertEqual(sat_product_count, kubectl_product_count)
+        self.assertGreaterEqual(sat_product_count, kubectl_product_count)
```

Comment:
A comment might be useful for later to understand why we are asserting there are an equal or greater number of products reported by `sat showrev` vs. the number of products in the main `cray-product-catalog` ConfigMap.